moped-toolbox/purchase_order_backfill/create_purchase_orders.py

```diff
@@ -0,0 +1,71 @@
+"""
+This script takes contractors and DO#s from existing moped projects and moves them into
+their own table
+
+Usage:
+
+"""
+
+import argparse
+
+
+from amd_milestones_backfill.utils import get_logger, make_hasura_request
+from queries import PROJECTS_QUERY, ADD_PURCHASE_ORDER
+from amd_milestones_backfill.secrets import HASURA
```

Comment:
I recognize that these are imports from a sibling toolbox folder, but because of that, I'm not able to get the script to run unmodified. I can symlink some stuff around to make it work, but I can't imagine that's what is intended. Please let me know if I'm missing something obvious 🙏

Reply:
Not missing anything obvious at all, I struggled with the imports. I decided to include the files in the same folder also keeping the different tools in the toolbox more isolated from each other.